game.py

```diff
@@ -14,12 +17,33 @@
 
 
 def snowman(snowman_word):
-    """Complete the snowman function
-    replace "pass" below with your own code
-    It should print 'Congratulations, you win!'
-    If the player wins and, 
-    'Sorry, you lose! The word was {snowman_word}' if the player loses
-    """
+
+    correct_letter_guess_statuses = build_letter_status_dict(snowman_word)
+    wrong_guesses_list = []
+
+    while len(wrong_guesses_list) < SNOWMAN_MAX_WRONG_GUESSES:
+        print_word_progress_string(snowman_word, correct_letter_guess_statuses)
+        print("Wrong guesses so far:", wrong_guesses_list)
+        print_snowman_graphic(len(wrong_guesses_list))
+
+        user_guess = get_letter_from_user(correct_letter_guess_statuses, wrong_guesses_list)
+
+        if user_guess in correct_letter_guess_statuses:
+            correct_letter_guess_statuses[user_guess] = True
+            print(f"Good Job! '{user_guess}' is in the word. \n ")
+
```

Comment:
I'd argue you don't need this blank line on line 34 because we don't need it to visually/logically separate the code on lines 31-33 and lines 35-37. 
```suggestion
```

Avoid adding unnecessary blank lines to your project because it makes your codebase longer and doesn't comply with the PEP8 style guide.

The guide says "Surround top-level function and class definitions with two blank lines." and "Extra blank lines may be used (sparingly) to separate groups of related functions."

Review the style guide recommendations on blank lines [here](https://peps.python.org/pep-0008/#blank-lines).